### PR TITLE
next steps page added

### PR DESCRIPTION
**Pull request by ccshah298**
# Description

Next Steps Page added.

Fixes # (issue)-111

# How to test?

Please provide a short summary how your changes can be tested?

# Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by rajanibusani — APPROVED** [review]

**Review by rajanibusani — COMMENTED** [review]
Good Job Charmi, Your next page is responsive in different screen sizes...

File: src/client/containers/NextStepsPage/NextStepsPage.styles.css [hunk 1]
```diff
@@ -0,0 +1,128 @@
+.main_part_nextPage {
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.styles.css:1]:
```suggestion
.main-part-next-page {
```

File: src/client/containers/NextStepsPage/NextStepsPage.styles.css [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  background-image: url(../../assets/images/Vector.svg);
+  width: 100vw;
+  height: 100vh;
+  font-family: 'Roboto', sans-serif;
+}
+
+.sidemenu_nextpage {
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.styles.css:8]:
```suggestion
.sidemenu-nextpage {
```

File: src/client/containers/NextStepsPage/NextStepsPage.styles.css [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+  top: 40%;
+  position: absolute;
+  z-index: 6;
+}
+
+.side_menu_div {
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.styles.css:14]:
```suggestion
.side-menu-div {
```

File: src/client/containers/NextStepsPage/NextStepsPage.styles.css [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+  background-color: #5088a8;
+  margin-left: 0%;
+}
+
+.orange_box {
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.styles.css:19]:
```suggestion
.orange-box {
```

File: src/client/containers/NextStepsPage/NextStepsPage.styles.css [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+  top: 50%;
+  left: 50%;
+  -ms-transform: translate(-50%, -50%);
+  transform: translate(-50%, -50%);
+  width: 545px;
+  height: 450px;
+  z-index: 2;
+}
+
+.white_box {
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.styles.css:33]:
```suggestion
.white-box {
```

File: src/client/containers/NextStepsPage/NextStepsPage.styles.css [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+  top: 50%;
+  left: 50%;
+  -ms-transform: translate(-50%, -50%);
+  transform: translate(-50%, -50%);
+  height: 380px;
+  width: 545px;
+  z-index: 3;
+}
+
+.circleImage {
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.styles.css:47]:
```suggestion
.circle-image {
```

File: src/client/containers/NextStepsPage/NextStepsPage.styles.css [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+  margin-top: 30px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  -ms-transform: translate(-50%, -50%);
+  transform: translate(-50%, -50%);
+  z-index: 4;
+}
+
+.cImage {
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.styles.css:58]:
```suggestion
.c-image {
```

File: src/client/containers/NextStepsPage/NextStepsPage.styles.css [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+  width: 250px;
+  height: 250px;
+}
+
+.white_box h1 {
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.styles.css:63]:
```suggestion
.white-box h1 {
```

File: src/client/containers/NextStepsPage/NextStepsPage.styles.css [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+  text-align: center;
+  color: #298f8a;
+  font-size: 1.4vw;
+}
+
+.middle_part {
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.styles.css:69]:
```suggestion
.middle-part {
```

File: src/client/containers/NextStepsPage/NextStepsPage.styles.css [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+  display: flex;
+}
+
+.mainImages {
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.styles.css:73]:
```suggestion
.main-images {
```

File: src/client/containers/NextStepsPage/NextStepsPage.styles.css [hunk 11]
```diff
@@ -0,0 +1,128 @@
...
+  display: flex;
+  justify-content: center;
+  margin: 0;
+  padding: 0;
+}
+
+.footerImages {
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.styles.css:80]:
```suggestion
.footer-images {
```

File: src/client/containers/NextStepsPage/NextStepsPage.styles.css [hunk 12]
```diff
@@ -0,0 +1,128 @@
...
+  display: flex;
+  justify-content: center;
+  padding: 0;
+  margin-top: 10px;
+  max-width: 480px;
+  list-style-type: none;
+}
+
+.footerImages li {
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.styles.css:89]:
```suggestion
.footer-images li {
```

File: src/client/containers/NextStepsPage/NextStepsPage.styles.css [hunk 13]
```diff
@@ -0,0 +1,128 @@
...
+  margin-right: -8px;
+}
+
+.rediImage,
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.styles.css:93]:
```suggestion
.redi-image,
```

File: src/client/containers/NextStepsPage/NextStepsPage.styles.css [hunk 14]
```diff
@@ -0,0 +1,128 @@
...
+.hyfImage {
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.styles.css:94]:
```suggestion
.hyf-image {
```

File: src/client/containers/NextStepsPage/NextStepsPage.styles.css [hunk 15]
```diff
@@ -0,0 +1,128 @@
...
+  display: flex;
+  padding: 0;
+  flex-direction: column;
+  vertical-align: middle;
+  text-align: center;
+}
+
+.hyfLogo {
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.styles.css:102]:
```suggestion
.hyf-logo {
```

File: src/client/containers/NextStepsPage/NextStepsPage.styles.css [hunk 16]
```diff
@@ -0,0 +1,128 @@
...
+  background-color: #293a7d;
+}
+
+.imageName {
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.styles.css:106]:
```suggestion
.image-name {
```

File: src/client/containers/NextStepsPage/NextStepsPage.styles.css [hunk 17]
```diff
@@ -0,0 +1,128 @@
...
+  font-size: 10px;
+  text-align: center;
+  display: block;
+  width: 90px;
+}
+
+.elevator_div {
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.styles.css:113]:
```suggestion
.elevator-div {
```

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 18]
```diff
@@ -1,3 +1,146 @@
...
 
-export const NextStepsPage = () => <section>Next Steps Page</section>;
+export const NextStepsPage = () => {
+  const history = useHistory();
+
+  function handleClick() {
+    history.push('/ElevatorPage');
+  }
+  return (
+    <div className="main_part_nextPage">
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:24]:
```suggestion
    <div className="main-part-nextPage">
```

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 19]
```diff
@@ -1,3 +1,146 @@
...
+      <div>
+        <PageHeader />
+      </div>
+      <div className="middle_part">
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:28]:
```suggestion
      <div className="middle-part">
```

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 20]
```diff
@@ -1,3 +1,146 @@
...
+        <div className="sidemenu_nextpage">
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:29]:
```suggestion
        <div className="sidemenu-nextpage">
```

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 21]
```diff
@@ -1,3 +1,146 @@
...
+          <SideMenu />
+        </div>
+        <div className="orange_box" />
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:32]:
```suggestion
        <div className="orange-box" />
```

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 22]
```diff
@@ -1,3 +1,146 @@
...
+        <div className="white_box">
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:33]:
```suggestion
        <div className="white-box">
```

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 23]
```diff
@@ -1,3 +1,146 @@
...
+          <h1>YOUR NEXT STEPS</h1>
+          <p>
+            <strong>Suggestions for Upgrading your Knowledge :</strong>
+          </p>
+          <p>Checkout our schools</p>
+          <div className="mainImages">
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:39]:
```suggestion
          <div className="main-images">
```

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 24]
```diff
@@ -1,3 +1,146 @@
...
+            <div className="rediImage">
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:40]:
```suggestion
            <div className="redi-image">
```

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 25]
```diff
@@ -1,3 +1,146 @@
...
+              <NavLink to="https://www.redi-school.org">
+                <img
+                  src={rediLogo}
+                  alt="redi-logo"
+                  width="50px"
+                  height="50px"
+                />
+              </NavLink>
+              <span className="imageName">Redi School</span>
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:49]:
```suggestion
              <span className="image-name">Redi School</span>
```

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 26]
```diff
@@ -1,3 +1,146 @@
...
+            </div>
+            &nbsp;
+            <div className="hyfImage">
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:52]:
```suggestion
            <div className="hyf-image">
```

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 27]
```diff
@@ -1,3 +1,146 @@
...
+              <NavLink to="https://www.hackyourfuture.dk">
+                <img
+                  className="hyfLogo"
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:55]:
```suggestion
                  className="hyf-logo"
```

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 28]
```diff
@@ -1,3 +1,146 @@
...
+                  src={hyfLogo}
+                  alt="hyf-logo"
+                  width="64px"
+                  height="50px"
+                />
+              </NavLink>
+              <span className="imageName">HackYourFuture School</span>
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:62]:
```suggestion
              <span className="image-name">HackYourFuture School</span>
```

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 29]
```diff
@@ -1,3 +1,146 @@
...
+            </div>
+          </div>
+          <p>
+            Or other website with some awsome{' '}
+            <strong>
+              Online <br /> courses like:
+            </strong>
+          </p>
+          <nav>
+            <ul className="footerImages">
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:72]:
```suggestion
            <ul className="footer-images">
```

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 30]
```diff
@@ -1,3 +1,146 @@
...
+                    width="111px"
+                    height="44px"
+                  />
+                </li>
+              </NavLink>
+            </ul>
+          </nav>
+        </div>
+      </div>
+      <div className="circleImage">
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:132]:
```suggestion
      <div className="circle-image">
```

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 31]
```diff
@@ -1,3 +1,146 @@
...
+        <img className="cImage" src={roundImg} alt="round-img" />
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:133]:
```suggestion
        <img className="c-image" src={roundImg} alt="round-img" />
```

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 32]
```diff
@@ -1,3 +1,146 @@
...
+      </div>
+      <div className="elevator_div">
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:135]:
```suggestion
      <div className="elevator-div">
```

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 33]
```diff
@@ -1,3 +1,146 @@
...
+        <Buttons
+          className="elevator_button"
```
Comment (bytewiz) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:137]:
```suggestion
          className="elevator-button"
```

**Review by bytewiz — CHANGES_REQUESTED** [review]

**Review by sofiiadidovych — APPROVED** [review]

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 34]
```diff
@@ -1,3 +1,145 @@
...
+              </NavLink>
+              <span className="image-name">Redi School</span>
+            </div>
+            &nbsp;
+            <div className="hyf-image-next-page">
+              <NavLink to="https://www.hackyourfuture.dk">
+                <img
+                  className="hyf-logo-next-page"
+                  src={hyfLogo}
+                  alt="hyf-logo"
```
Comment (dpfernandes) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:57]: This is not a classname so you can write HackYourFuture Copenhagen instead

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 35]
```diff
@@ -1,3 +1,145 @@
...
+              Online <br /> courses like:
+            </strong>
+          </p>
+          <nav>
+            <ul className="footer-images">
+              <NavLink to="https://www.udacity.com/blog/category/free-courses">
+                <li>
+                  <img
+                    src={udacityImg}
+                    alt="udacity-img"
```
Comment (dpfernandes) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:77]: This is not a classname so you can write Udacity instead

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 36]
```diff
@@ -1,3 +1,145 @@
...
+                    width="75px"
+                    height="50px"
+                  />
+                </li>
+              </NavLink>
+              <NavLink to="https://www.coursera.org">
+                <li>
+                  <img
+                    src={courseraImg}
+                    alt="coursera-img"
```
Comment (dpfernandes) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:87]: This is not a classname so you can write Coursera instead

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 37]
```diff
@@ -1,3 +1,145 @@
...
+                    width="50px"
+                    height="50px"
+                  />
+                </li>
+              </NavLink>
+              <NavLink to="https://www.linkedin.com/">
+                <li>
+                  <img
+                    src={linkedinImg}
+                    alt="linkedin-img"
```
Comment (dpfernandes) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:97]: This is not a classname so you can write LinkedIn instead

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 38]
```diff
@@ -1,3 +1,145 @@
...
+                    width="165px"
+                    height="25px"
+                  />
+                </li>
+              </NavLink>
+              <NavLink to="https://www.lynda.com">
+                <li>
+                  <img
+                    src={lyndaImg}
+                    alt="lynda-img"
```
Comment (dpfernandes) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:107]: This is not a classname so you can write Lynda instead

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 39]
```diff
@@ -1,3 +1,145 @@
...
+              <NavLink to="https://careerfoundry.com">
+                <li>
+                  <img src={cfImg} alt="cf-img" width="50px" height="50px" />
+                </li>
+              </NavLink>
+              <NavLink to="https://www.udemy.com">
+                <li>
+                  <img
+                    src={udemyImg}
+                    alt="udemy-img"
```
Comment (dpfernandes) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:122]: This is not a classname so you can write Udemy instead

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 40]
```diff
@@ -1,3 +1,145 @@
...
+                    src={lyndaImg}
+                    alt="lynda-img"
+                    width="50px"
+                    height="50px"
+                  />
+                </li>
+              </NavLink>
+              <NavLink to="https://careerfoundry.com">
+                <li>
+                  <img src={cfImg} alt="cf-img" width="50px" height="50px" />
```
Comment (dpfernandes) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:115]: On the Alt value you can write Career Foundry instead

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 41]
```diff
@@ -1,3 +1,145 @@
...
+                    height="44px"
+                  />
+                </li>
+              </NavLink>
+            </ul>
+          </nav>
+        </div>
+      </div>
+      <div className="circle-image">
+        <img className="c-image" src={roundImg} alt="round-img" />
```
Comment (dpfernandes) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:133]: The Alt value is not a Classname so you can write "Guide IT"

**Review by dpfernandes — CHANGES_REQUESTED** [review]
Please replace the alt values with text that describes what the image is

File: src/client/containers/NextStepsPage/NextStepsPage.component.js (hunk 41 above)
Comment (ccshah298) [reply src/client/containers/NextStepsPage/NextStepsPage.component.js]: Thanks for review @dpfernandes .. its done as per your comment.. :)

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 42]
```diff
@@ -1,3 +1,140 @@
...
+          <p>
+            <strong>Suggestions for Upgrading your Knowledge :</strong>
+          </p>
+          <p>Checkout our schools</p>
+          <div className="main-images">
+            <div className="redi-image-next-page">
+              <NavLink to="https://www.redi-school.org">
+                <img
+                  src={rediLogo}
+                  alt="Redischool"
```
Comment (dpfernandes) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:44]: The alt value can have spaces so please write the name of the school which is ReDI School

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 43]
```diff
@@ -1,3 +1,140 @@
...
+                  width="50px"
+                  height="50px"
+                />
+              </NavLink>
+              <span className="image-name">Redi School</span>
```
Comment (dpfernandes) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:49]: It's called ReDI School

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 44]
```diff
@@ -1,3 +1,140 @@
...
+            </div>
+            &nbsp;
+            <div className="hyf-image-next-page">
+              <NavLink to="https://www.hackyourfuture.dk">
+                <img
+                  className="hyf-logo-next-page"
+                  src={hyfLogo}
+                  alt="HackYourFuture"
```
Comment (dpfernandes) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:57]: HackYourFuture Copenhagen

File: src/client/containers/NextStepsPage/NextStepsPage.component.js [hunk 45]
```diff
@@ -1,3 +1,140 @@
...
+                    width="50px"
+                    height="50px"
+                  />
+                </li>
+              </NavLink>
+              <NavLink to="https://www.linkedin.com/">
+                <li>
+                  <img
+                    src={linkedinImg}
+                    alt="Linkedin"
```
Comment (dpfernandes) [inline src/client/containers/NextStepsPage/NextStepsPage.component.js:97]: LinkedIn - Capital i

**Review by dpfernandes — CHANGES_REQUESTED** [review]

File: src/client/containers/NextStepsPage/NextStepsPage.component.js (hunk 45 above)
Comment (ccshah298) [reply src/client/containers/NextStepsPage/NextStepsPage.component.js]: done ✅

**Review by dpfernandes — APPROVED** [review]
👍

**Pull request merged**